src/TestConsoleApp/TestConsoleApp.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\TestApp\TestApp\TestApp.csproj" />
+    <ProjectReference Include="..\Xablu.WebApiClient\Xablu.WebApiClient.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Mono.Android">
```

Comment:
How comes that a project targeting .NET Core has a reference to Mono.Android? 🤔 was it added by accident? Can we get rid of it?